package.json

```diff
@@ -16,7 +16,7 @@
     "node": ">= 0.8.x"
   },
   "scripts": {
-    "download": "curl \"http://unicode.org/Public/9.0.0/ucd/UnicodeData.txt\" > UnicodeData.txt",
+    "download": "curl \"https://unicode.org/Public/9.0.0/ucd/UnicodeData.txt\" > UnicodeData.txt",
```

Comment:
No, unfortunately the URL is blocked for me. I naively assumed a site like this would support HTTPS these days...